www/include/configuration/configKnowledge/display-hostTemplates.php

```diff
@@ -111,22 +101,26 @@
         "WHERE host.host_id = ehi.host_host_id " .
         "AND host.host_register = '0' " .
         "AND host.host_locked = '0' ";
-    if (isset($_REQUEST['searchHostTemplate']) && $_REQUEST['searchHostTemplate']) {
-        $query .= " AND host.host_name LIKE '%" . $_REQUEST['searchHostTemplate'] . "%' ";
+    if (!empty($postHostTemplate)) {
+        $query .= "AND host.host_name LIKE :postHostTemplate ";
+    }
+    $query .= "ORDER BY " . $orderby . " " . $order . " LIMIT " . $num * $limit . ", " . $limit;
```

Comment:
order by is harcoded... as you removed user's filter
https://github.com/centreon/centreon/pull/8815/files#diff-e8d3d31415f23da2771e17637d4355f9R52

Reply:
there is only one value available for orderBy => host_name this is why it is hardcoded instead of getting this value in $_POST. But it could be revert if needed.

Reply:
$num * $limit and $limit are now binded but i kept the $orderBy variable instead of writing directly his value because it is use later in the file